src/bentoml/_internal/bento/bento.py

```diff
@@ -370,6 +381,8 @@ class BentoRunnerInfo:
 
     @classmethod
     def from_runner(cls, r: Runner) -> BentoRunnerInfo:
+        if LazyType["TritonRunner"]("bentoml.triton.Runner").isinstance(r):
+            return cls(name=r.name, runnable_type=r.runnable_class.__name__)
```

Comment:
Is this necessary? Shouldn't the triton runner just have an empty list for `models` and `None` for config?